### PR TITLE
FEATURE: QA-17800 automation test for pdf export

### PR DESCRIPTION
**Pull request by KhangNguyenlhv**
---

Supported PR commands:

| Command                                                 | Description                                                |
| ------------------------------------------------------- | ---------------------------------------------------------- |
| `ok to test`                                            | Re-run standard checks                                     |
| `extended check sonar`                                  | SonarQube tests                                            |
| `extended test - backstop`                              | BackstopJS tests                                           |
| **E2E Cypress tests commands - TIGER**                  |                                                            |
| `extended test - tiger-cypress - isolated <testName>`   | Run isolated tests running against recorded Tiger backend. |
| `extended test - tiger-cypress - record <testName>`     | Create a new recording for isolated Tiger tests.           |
| `extended test - tiger-cypress - integrated <testName>` | Run integrated tests against live backend                  |
| **E2E Cypress tests commands - BEAR**                   |                                                            |
| `extended test - cypress - isolated <testName>`         | Run isolated tests running against recorded Bear backend.  |
| `extended test - cypress - record <testName>`           | Create a new recording for isolated Bear tests.            |
| `extended test - cypress - integrated <testName>`       | Run integrated tests against live backend                  |

`<testName>` in cypress commands is used to filter specfiles. Example, to run record with BEAR backend

-   Against `dashboard.spec.ts` and `drilling.spec.ts`, execute command `extended test - cypress - record dashboard,drilling`
-   Against all specfiles, execute command `extended test - cypress - record` or `extended test - cypress - record *`

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `extended test - backstop` passes
-   [ ] `extended test - tiger-cypress - record` to record new mapping files (Tiger BE)
-   [ ] `extended test - cypress - record` to record new mapping files (Bear BE)
-   [ ] `extended test - tiger-cypress - isolated` passes
-   [ ] `extended test - cypress - isolated` passes
-   [ ] `extended test - tiger-cypress - integrated` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12204/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14169/ : FAILURE in 38s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12205/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14170/ : FAILURE in 40s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12206/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14171/ : FAILURE in 33s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12207/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14172/ : FAILURE in 38s

**Comment by KhangNguyenlhv** [comment]
extended test - tiger-cypress - integrated exportDashboardToPDF.spec.ts

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12208/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/213/ : FAILURE in 46s

**Comment by KhangNguyenlhv** [comment]
extended test - tiger-cypress - isolated exportDashboardToPDF.spec.ts

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12209/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/214/ : FAILURE in 45s

**Comment by KhangNguyenlhv** [comment]
extended test - backstop

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12210/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/215/ : FAILURE in 59s

**Comment by KhangNguyenlhv** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12211/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14173/ : FAILURE in 33s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12212/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14174/ : FAILURE in 35s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12213/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14175/ : FAILURE in 34s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12215/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14177/ : FAILURE in 5m 53s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12219/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14182/ : FAILURE in 9m 35s

**Comment by gdgate** [comment]
Build in pipeline check-extended aborted.

**Comment by KhangNguyenlhv** [comment]
extended test - cypress - isolated exportDashboardToPDF.spec.ts

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12230/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14190/ : SUCCESS in 18m 43s

**Comment by KhangNguyenlhv** [comment]
extended test - tiger-cypress - integrated

**Comment by KhangNguyenlhv** [comment]
extended test - tiger-cypress - isolated

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12231/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/218/ : SUCCESS in 39m 11s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12247/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14207/ : FAILURE in 51s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12248/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14208/ : FAILURE in 38s

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12249/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14209/ : SUCCESS in 15m 12s

File: libs/sdk-ui-tests-e2e/cypress/tools/export.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// (C) 2022-2023 GoodData Corporation
+import { join } from "path";
```
Comment (hoaithuong) [inline libs/sdk-ui-tests-e2e/cypress/tools/export.ts:2]: Please add an empty line before import.

File: libs/sdk-ui-tests-e2e/cypress/plugins/readPdf.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// (C) 2022-2023 GoodData Corporation
+import * as fs from "fs";
```
Comment (hoaithuong) [inline libs/sdk-ui-tests-e2e/cypress/plugins/readPdf.ts:2]: Please add an empty line before import.

File: libs/sdk-ui-tests-e2e/cypress/fixtures/dashboardInfosForExport.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+    "validInsightsForTigerPDFExport": [
+        {
+            "dashboardTitle": "KD with compares",
+            "dashboardURL": "dashboard/compare",
+            "fileName": "KD with compares.pdf",
+            "contents": "Region\nEast CoastWest Coast\nStage Name\n\nSum of Amount: Stage Name (5)Sum of Amount: Stage Name (5)\nInterest112,261,028.81658,375,577.02\nDiscovery81,789,169.68181,439,545.32\nShort List–219,302,451.65\nAD with compares\nKD with compares"
```
Comment (hoaithuong) [inline libs/sdk-ui-tests-e2e/cypress/fixtures/dashboardInfosForExport.json:7]: What does "\n" mean in the contents?

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/exportDashboardToPDF.spec.ts [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+const dashboardMenu = new DashboardMenu();
+const productFilter = new AttributeFilter("Product");
+const exportControl = new Export();
+const widget = new Widget(0);
+const topBar = new TopBar();
+const NULL_KD_URL = "dashboard/nullvalue";
+const NULL_KD_TITLE = "KD has null value";
+const NULL_KD_FILE_NAME = "KD has null value.pdf";
+const NULL_KD_CONTENS =
```
Comment (hoaithuong) [inline libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/exportDashboardToPDF.spec.ts:29]: Please move all of const to above the Cypress.Cookies.defaults({.

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/exportDashboardToPDF.spec.ts [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    "Region\n(empty value)\nProductMetric has null value_Snapshot [EOP]_Timeline [EOP]\nWonderKid24,09641,054.0044,195\nAD has null value\nKD has null value";
+
+describe("Export dashboard to pdf", () => {
+    beforeEach(() => {
+        cy.login();
+    });
+
+    it(
+        "should export insight to PDF from dashboards",
+        { tags: ["checklist_integrated_tiger", "pre-merge_isolated_tiger"] },
```
Comment (hoaithuong) [inline libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/exportDashboardToPDF.spec.ts:39]: Only input "checklist_integrated_tiger" for the tag.

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/exportDashboardToPDF.spec.ts [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+                        exportControl.expectExportedPDF(dashboardInfo.fileName, dashboardInfo.contents);
+                    },
+                );
+            });
+        },
+    );
+
+    it(
+        "is able to export dashboard with temporary filter to pdf",
+        { tags: ["checklist_integrated_tiger", "pre-merge_isolated_tiger"] },
```
Comment (hoaithuong) [inline libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/exportDashboardToPDF.spec.ts:61]: Same above, and move the tag into "describe" if these are the same tag.

File: libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Dashboard/DashboardHasNullValueScenario.tsx [hunk 7]
```diff
@@ -0,0 +1,10 @@
+// (C) 2021-2023 GoodData Corporation
+import React from "react";
```
Comment (hoaithuong) [inline libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Dashboard/DashboardHasNullValueScenario.tsx:2]: Same above.

File: libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Dashboard/DashboardParentScenario.tsx [hunk 8]
```diff
@@ -0,0 +1,10 @@
+// (C) 2021-2023 GoodData Corporation
+import React from "react";
```
Comment (hoaithuong) [inline libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Dashboard/DashboardParentScenario.tsx:2]: Same above.

File: libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Dashboard/DashboardWithCompareScenario.tsx [hunk 9]
```diff
@@ -0,0 +1,10 @@
+// (C) 2021-2023 GoodData Corporation
+import React from "react";
```
Comment (hoaithuong) [inline libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Dashboard/DashboardWithCompareScenario.tsx:2]: Same above.

File: libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Dashboard/DashboardWithInvalidInsightScenario.tsx [hunk 10]
```diff
@@ -0,0 +1,10 @@
+// (C) 2021-2023 GoodData Corporation
+import React from "react";
```
Comment (hoaithuong) [inline libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Dashboard/DashboardWithInvalidInsightScenario.tsx:2]: Same above.

File: libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Dashboard/DashboardWithManyDataScenario.tsx [hunk 11]
```diff
@@ -0,0 +1,10 @@
+// (C) 2021-2023 GoodData Corporation
+import React from "react";
```
Comment (hoaithuong) [inline libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Dashboard/DashboardWithManyDataScenario.tsx:2]: Same above.

File: libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Dashboard/DashboardWithMergeAndUnmergeScenario.tsx [hunk 12]
```diff
@@ -0,0 +1,10 @@
+// (C) 2021-2023 GoodData Corporation
+import React from "react";
```
Comment (hoaithuong) [inline libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Dashboard/DashboardWithMergeAndUnmergeScenario.tsx:2]: Same above.

File: libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Dashboard/DashboardWithNoDataScenario.tsx [hunk 13]
```diff
@@ -0,0 +1,10 @@
+// (C) 2021-2023 GoodData Corporation
+import React from "react";
```
Comment (hoaithuong) [inline libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Dashboard/DashboardWithNoDataScenario.tsx:2]: Same above.

File: libs/sdk-ui-tests-e2e/cypress/fixtures/dashboardInfosForExport.json (hunk 3 above)
Comment (KhangNguyenlhv) [reply libs/sdk-ui-tests-e2e/cypress/fixtures/dashboardInfosForExport.json:7]: It's just blank space. But it has nothing to do with the content of the pdf file. I removed it.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12272/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14220/ : SUCCESS in 15m 30s

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/exportDashboardToPDF.spec.ts [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+const dashboardMenu = new DashboardMenu();
+const productFilter = new AttributeFilter("Product");
+const exportControl = new Export();
+const widget = new Widget(0);
+const topBar = new TopBar();
+const NULL_KD_URL = "dashboard/nullvalue";
+const NULL_KD_TITLE = "KD has null value";
+const NULL_KD_FILE_NAME = "KD has null value.pdf";
+const NULL_KD_CONTENS =
+    "Region\n(empty value)\nProductMetric has null value_Snapshot [EOP]_Timeline [EOP]\nWonderKid24,09641,054.0044,195\nAD has null value\nKD has null value";
```
Comment (hoaithuong) [inline libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/exportDashboardToPDF.spec.ts:19]: Please move NULL_KD const to your Json too.

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/exportDashboardToPDF.spec.ts [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+});
+
+Cypress.on("uncaught:exception", (error) => {
+    console.error("Uncaught exception cause", error);
+    return false;
+});
+
+Cypress.Cookies.debug(true);
+
+describe("Export dashboard to pdf", { tags: ["checklist_integrated_tiger"] }, () => {
```
Comment (hoaithuong) [inline libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/exportDashboardToPDF.spec.ts:32]: Remove [] of the tag if your tag has only one.

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12318/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14249/ : SUCCESS in 15m 12s

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12319/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14251/ : FAILURE in 37s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12320/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14252/ : FAILURE in 36s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12321/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14253/ : FAILURE in 35s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12322/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14254/ : SUCCESS in 15m 24s

**Comment by KhangNguyenlhv** [comment]
extended test - tiger-cypress - isolated

**Comment by KhangNguyenlhv** [comment]
extended test - tiger-cypress - integrated

**Comment by KhangNguyenlhv** [comment]
extended test - tiger-cypress - integrated exportDashboardToPDF.spec.ts

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12324/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/260/ : SUCCESS in 12m 21s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12323/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14255/ : SUCCESS in 18m 22s

**Comment by KhangNguyenlhv** [comment]
extended test - tiger-cypress - integrated exportDashboardToPDF.spec.ts

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12325/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/261/ : FAILURE in 5m 03s

**Comment by KhangNguyenlhv** [comment]
extended test - tiger-cypress - integrated headerSection.spec.ts

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12329/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/266/ : FAILURE in 5m 21s

**Comment by KhangNguyenlhv** [comment]
extended test - tiger-cypress - integrated

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12330/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/267/ : FAILURE in 6m 31s

**Comment by KhangNguyenlhv** [comment]
extended test - tiger-cypress - isolated

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12331/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/268/ : SUCCESS in 8m 20s

**Comment by KhangNguyenlhv** [comment]
extended test - tiger-cypress - integrated

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12332/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/269/ : FAILURE in 7m 14s

**Review by hoaithuong — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12344/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14262/ : FAILURE in 33s

**Comment by KhangNguyenlhv** [comment]
extended test - tiger-cypress - isolated

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12345/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/275/ : UNSTABLE in 1m 00s

**Review by pbenes — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3603

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2576/ : SUCCESS in 14m 52s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/458/ : FAILURE in 1m 33s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14264/ : FAILURE in 41s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/16/ : UNSTABLE in 40s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/45/ : UNSTABLE in 1m 11s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/3732/ : SUCCESS in 17s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3603

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2582/ : SUCCESS in 11m 54s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/465/ : FAILURE in 2m 31s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14281/ : FAILURE in 55s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/25/ : UNSTABLE in 6m 41s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/53/ : UNSTABLE in 7m 26s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/3738/ : SUCCESS in 13s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3603

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2583/ : SUCCESS in 11m 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/466/ : FAILURE in 2m 25s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14284/ : FAILURE in 43s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/27/ : UNSTABLE in 17m 12s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/55/ : UNSTABLE in 18m 08s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/3739/ : SUCCESS in 16s

**Comment by KhangNguyenlhv** [comment]
extended test - tiger-cypress - isolated

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12360/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14287/ : SUCCESS in 23m 41s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12361/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/293/ : SUCCESS in 8m 16s

**Comment by KhangNguyenlhv** [comment]
extended test - tiger-cypress - isolated

**Comment by KhangNguyenlhv** [comment]
extended test - tiger-cypress - integrated

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12362/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/294/ : SUCCESS in 8m 52s

**Review by hungnguyenauto — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3603

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2584/ : SUCCESS in 11m 07s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/467/ : FAILURE in 2m 51s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14290/ : SUCCESS in 15m 51s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/31/ : SUCCESS in 9m 21s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/57/ : UNSTABLE in 10m 53s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/3740/ : SUCCESS in 13s

**Pull request merged**